db/rdbms/migration/0002_migrate_descriptor_to_extended_descriptor.go

```diff
@@ -58,21 +60,32 @@ const shardSize = uint64(50)
 // jobs table.
 type DescriptorMigration struct {
 	log *logrus.Entry
+	db  *sql.DB
+}
+
+type dbConn interface {
+	Exec(query string, args ...interface{}) (sql.Result, error)
+	Query(query string, args ...interface{}) (*sql.Rows, error)
+}
```

Comment:
I will probably move this somewhere else. It is actually an interface implemented also in the rdbms plugin, but I'd rather duplicate, than introduce a dependency between the two.